.github/workflows/check.yml

```diff
@@ -0,0 +1,34 @@
+name: Helm Chart CI
+
+on:
+  push:
+
+jobs:
+  helm_chart_ci:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Step 1: Checkout the repository
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      # Step 2: Set up Helm
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+
+      # Step 3: Lint the Helm chart
+      - name: Lint Helm chart
+        run: |
+          helm lint .
+
+      # Step 4: Install kubeconform
+      - name: Install kubeconform
+        run: |
+          curl -LO https://github.com/yannh/kubeconform/releases/latest/download/kubeconform-linux-amd64.tar.gz
+          tar -xzf kubeconform-linux-amd64.tar.gz
+          sudo mv kubeconform /usr/local/bin/
```

Comment:
**Nitpick** ⛏️ Rather than doing a privileged operation to move the executable to a protected directory, this could remain in the current working directory and be invoked with `./kubeconform`.

Reply:
I can get behind this. I wasn't concerned because the run fires in an encapsulated environment but that would be a long term assumption that it remains true forever 😁

Reply:
Great idea. This comment is already valuable if only for sharing the thought behind it. 💪